App/Views/Employe/TableauBord.php

```diff
@@ -0,0 +1,176 @@
+<?php declare(strict_types = 1);
+
+/* 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Whitespace found at end of line](https://app.codacy.com/app/Libertempo/web/pullRequest?prid=4406329)